src/persist/src/cfg.rs

```diff
@@ -148,6 +152,34 @@ impl BlobConfig {
                 query_params.clear();
                 Ok(BlobConfig::Mem(tombstone))
             }
+            "http" | "https" => match url.host().expect("hostname").to_string().split_once('.') {
```

Comment:
Mind turning these `expect`s into `ExternalError`s? `anyhow` is fine. But it would be a shame to bring down MZ over a malformed string here...

Reply:
fixed!